src/styl/responsive/mobile.styl

```diff
@@ -23,23 +23,33 @@
 
 #what-is-video-player
   position: relative
+  transition: all $transition-speed-fast
+
   &:hover
+    transform: scale(1.02)
```

Comment:
I think this is backwards. The initial state should be reduced from 100% so that when playing the pixels are not being stretched